app/controllers/trip_families_controller.rb

```diff
@@ -0,0 +1,40 @@
+class TripFamiliesController < ApplicationController
+  def index
+    @trip_families = trip.trip_families.all
+  end
+
+  def show
+    @trip_family = trip.trip_families.find(params[:id])
+
+    respond_with @trip_family, location: -> { trip_family_path }
```

Comment:
I think this line can be removed completely

Reply:
Done in sha `c8fc6f2`